vaadin-select-flow-parent/vaadin-select-flow/src/test/java/com/vaadin/flow/component/select/validation/BasicValidationTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.select.validation;
+
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.tests.validation.AbstractBasicValidationTest;
+
+public class BasicValidationTest
```

Comment:
Looks like `Select` is missing a Binder validation test?

Reply:
It is not needed as Select doesn't implement the `getDefaultValidator` method.